blueocean-rest/README.md

```diff
@@ -337,6 +337,22 @@ Pipelines can be nested inside folder.
       "qeueudTime" : "2016-06-22T11:05:41.309+1200"
     }
 
+## Replay a pipeline run
```

Comment:
Can it mention here if it runs the exact commit or HEAD? (I have already forgotten)